### ISSUE TITLE
Swipe to bring up ToC

### ISSUE THREAD

**Issue by Inbefortus** [kiwix/kiwix-js-windows#191]
For tablet users with a large screen, it would be easier if the Table of Contents could be highlighted by swiping. This should be achieved through a setting.

Like this:

![20210909_154527](https://user-images.githubusercontent.com/71934042/132698580-8f79dc7c-c3d3-4183-8c39-080a01b135f2.gif)

In the long-term use, this would be a great help!

**Comment by Jaifroid** [comment]
@Inbefortus Would this be a touch-only feature? It really depends on whether there is a "swipe" gesture that can be coded easily and reliably in JavaScript (I'm pretty sure there isn't a native JS swipe "event"). My suspicion is that it would require a framework that supports swipe. Trying to do it natively in JS looks fiendishly complicated (see https://css-tricks.com/simple-swipe-with-vanilla-javascript/ , and see the amount of code involved as you scroll down the tutorial...). It's a nice idea.

**Comment by Inbefortus** [comment]
@Jaifroid Yes, it should be particularly the same gesture as in the other Kiwix Android app where you swipe left on the edge of the screen, you can see that well above (GIF).

I know, we people who have special requests have easy talk, implementation is a whole other thing. I would say that's up to you to decide. If you think it's worth the work, then there's no harm in implementing it. Like I said, it would be a relief for users like me. :)

**Comment by Jaifroid** [comment]
This doesn't seem to be possible with current JS display frameworks.
